docs/rackhd/install_os.rst

```diff
@@ -0,0 +1,128 @@
+rootSshKey          String           *optional*   The SSH key that will be appended to target OS.
+installDisk         String/Number    *optional*   *installDisk* is to specify the target disk which the OS will be installed on. It can be a string or a number. **For string**, it is a disk path that the OS can recongize, its format varies with OS. For example, "/dev/sda" or "/dev/disk/by-id/scsi-36001636121940cc01df404d80c1e761e" for CentOS/RHEL, "t10.ATA_____SATADOM2DSV_3SE__________________________20130522AA0990120088" or "naa.6001636101840bb01df404d80c2d76fe" or "mpx.vmhba1:C0:T1:L0" or "vml.0000000000766d686261313a313a30" for ESXi. **For number**, it is a RackHD generated disk identifier (it could be obtained from "driveId" catalog). If *installDisk* is omitted, RackHD will assign the default disk by order: SATADOM -> first disk in "driveId" catalog -> "sda" for Linux OS or "firstdisk" for ESXi.
+kvm                 Boolean          *optional*   The value is *true* or *false* to indicates to install kvm or not, default is **false**.
+switchDevices       Array            *optional*   **(ESXi only)** If specified, this contains an array of objects with switchName and uplinks (optional) parameters.  If *uplinks* is omitted, null or empty, the vswitch will be created with no uplinks.
+postInstallCommands Array            *optional*   **(ESXi only)** If specified, this contains an array of string commands that will be run at the end of the post installation step.  This can be used by the customer to tweak final system configuration.
+
+=================== ================ ============ ============================================
+
+For **users** in payload:
+
+=========== ======== ============ ============================================
+Parameters  Type     Flags        Description
+=========== ======== ============ ============================================
+name        String   **required** The name of user
+password    String   **required** The password of user, it could be clear text, RackHD will do encryption before store it into kickstart file.
+uid         Number   **required** The unique identifier of user. **required** for non-ESXi OS, Not support for ESXi OS
+sshKey      String   *optional*   The SSH key that will be appended into target OS.
+=========== ======== ============ ============================================
+
+
+For **networkDevices** in payload, both ipv4 and ipv6 are supported
+
+============== ======== ============ ============================================
+Parameters     Type     Flags        Description
+============== ======== ============ ============================================
+device         String   **required**  Network device name (ex. "eth0")
+ipv4           Object   **required**  The object contains ipv4 configurations
+ipv6           Object   **required**  The object contains ipv6 configurations
```

Comment:
@pengz1 They are not a must.  In ESXi, if both are absent than the interface is set to DHCP.  In centos, they are either/or such that either ipv4 can be defined or ipv6 can be defined.